authors/apps/articles/views.py

```diff
@@ -0,0 +1,109 @@
+import os
+import django
+import json
+import random
+
+from rest_framework import status, exceptions
+from rest_framework import generics
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from datetime import datetime, timedelta
+from rest_framework.views import Response
+from django.template.defaultfilters import slugify
+
+from .renderers import ArticleJSONRenderer
+from .models import Article
+from .serializers import ArticleSerializer
+from .messages import error_msgs, success_msg
+
+
+class ArticleAPIView(generics.ListCreateAPIView):
+    """
+        Article endpoints
+    """
+    # renderer_classes = (ArticleJSONRenderer,)
+    queryset = Article.objects.all()
+    serializer_class = ArticleSerializer
+
+    def post(self, request):
+        """
+            POST /api/v1/articles/
+        """
+        permission_classes = (IsAuthenticated,)
+        article = request.data.get("article", {})
+        context = {"request": request}
+        a_slug = slugify(request.data['article']['title'])
+        author = request.user.id
+        random_number = 1
+        unique_slug = a_slug
```

Comment:
@danuluma slugs are better created in the models as opposed to views

Reply:
Also it would be nice if you could include the like dislike count in the articles view

Reply:
Creating articles was @tesh254's task. I haven't merged the latest changes from his branch because it's still WIP